indolaw-nextjs/pages/index.js

```diff
@@ -1,11 +1,27 @@
 import Head from "next/head";
 import Link from "next/link";
 import fs from "fs";
-import { GetStaticProps } from "next";
-import styles from "../styles/Home.module.css";
+import { useState } from "react";
 
+// TODO willem:
+// 1. Search bar (fuzzy search + uu number search)
 export default function Home(props) {
   const { laws } = props.data;
+  const ALL_YEAR_STRING = "Show All";
+
+  const [selectedYear, setSelectedYear] = useState(ALL_YEAR_STRING);
+  const [selectedLaws, setSelectedLaws] = useState(laws);
```

Comment:
do we really need this state variable? IIRC, seems like it's just a pure function of `selectedYear` & `laws` right?